apps/omg_watcher_info/test/support/factory.ex

```diff
@@ -58,12 +58,69 @@ defmodule OMG.WatcherInfo.Factory do
 
   alias OMG.WatcherInfo.DB
 
+  @eth OMG.Eth.RootChain.eth_pseudo_address()
+
+  @doc """
+  Block factory.
+
+  Generates a block in an incremental blknum of 1000, 2000, 3000, etc.
+  """
   def block_factory() do
     %DB.Block{
       blknum: sequence(:block_blknum, fn seq -> seq * 1000 end),
-      hash: sequence(:block_hash, fn seq -> <<seq::256>> end),
+      hash: insecure_random_bytes(32),
```

Comment:
I recommend changing back to using `sequence()` here.
```suggestion
      hash: sequence(:block_hash, fn seq -> <<seq::256>> end),
```

Reply:
Can you explain your rationale a bit?

Reply:
Nevermind I saw your comment after!